### PR TITLE
ARD-10456 switch to ardoq/leiningen jre base image

### PR DESCRIPTION
**Pull request by goroi**

### CODE REVIEW

**Review by ianmuge — APPROVED** [review]

File: project.clj [hunk 1]
```diff
@@ -16,18 +16,17 @@
             [com.ebaxt.lein-ver "1.2.0"]]
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [compojure "1.6.0"]
-                 [clj-http "3.7.0" :exclusions [com.fasterxml.jackson.core/jackson-databind]]
+                 [clj-http "3.12.2"]
```
Comment (mslupny) [inline project.clj:19]: Just curious: what's the reason for removing exclusions? Do you know why it was there in the first place?

**Review by mslupny — APPROVED** [review]

File: project.clj (hunk 1 above)
Comment (goroi) [reply project.clj:19]: I think @ianmuge added them and then included a specific version explicitly ...
but now with the newest version of cheshire the dependency is resolved to jackson .... 2.12.4

Comment (ianmuge) [reply project.clj:19]: It was there before I got to it last week

Comment (goroi) [reply project.clj:19]: hmhmh actually I think i am wrong I now see that it was Kristian Fri Jan 26 09:31:02 2018 that added it logn time ago.

Comment (goroi) [reply project.clj]: restoring it back then... just to be on the safe side.

**Pull request merged**